src/rf/apps/core/models.py

```diff
@@ -222,16 +232,10 @@ class LayerImage(Model):
         default=0,
         help_text='The order which images are layered (starting from 0)'
     )
-    thumb_small = URLField(
-        null=True,
-        blank=True,
-        help_text='80x80 pixels',
-    )
-    thumb_large = URLField(
-        null=True,
-        blank=True,
-        help_text='300x300 pixels',
-    )
+    thumb_small_key = CharField(max_length=255, blank=True, default='',
```

Comment:
I think we'll have to add a new column if we ever want to change these dimensions (once the application is live). Otherwise, we'll have to distort all the previously generated thumbnails to fit the new size. I would be in favor of making the dimensions part of the column name.